c/src/proactor/win_iocp.c

```diff
@@ -3427,11 +3414,9 @@ const pn_netaddr_t *pn_listener_addr(pn_listener_t *l) {
 }
 
 pn_millis_t pn_proactor_now(void) {
-  FILETIME now;
-  GetSystemTimeAsFileTime(&now);
-  ULARGE_INTEGER t;
-  t.u.HighPart = now.dwHighDateTime;
-  t.u.LowPart = now.dwLowDateTime;
-  // Convert to milliseconds and adjust base epoch
-  return t.QuadPart / 10000 - 11644473600000;
+    return (pn_millis_t) pn_proactor_now_64();
+}
+
+int64_t pn_proactor_now_64(void) {
```

Comment:
@astitcher So you you think that pn_proactor_now whould return signed or unsigned value? Because one response given your comments could be to just go unsigned everywhere.

Reply:
Thanks for the explanation. I am going to make the change.